### PR TITLE
Atom 4550 reauthorization requests

### PR DESCRIPTION
**Pull request by captainbeardo**

### CODE REVIEW

File: lib/conduit/marben/actions/reauthorization_request.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'conduit/marben/actions/base'
+
+module Conduit::Driver::Marben
+  class ReauthorizationRequest < Base
+    required_attributes :message_ete, :message_hbh, :session_id,
+      :origin_host, :origin_realm, :destination_realm, :destination_host
+
+    def initialize(options = {})
+      super # thanks for asking...
```
Comment (pedromartinez) [inline lib/conduit/marben/actions/reauthorization_request.rb]: huh?

File: lib/conduit/marben/views/reauthorization_request.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <avp name="Destination-Realm" type="DiameterIdentity" code="296">
+    <avp-value-simple str="<%= destination_realm %>" />
+    <avp-flags vendor-specific="no" protected="no" mandatory="yes" />
+  </avp>
+  <avp code="258" name="Auth-Application-Id" type="Unsigned32">
+    <avp-value-simple unsigned32="4" hexanum="0x4" />
+    <avp-flags vendor-specific="no" protected="no" mandatory="yes" />
+  </avp>
+  <avp name="Re-Auth-Request-Type" type="Unsigned32" code="258">
+    <avp-value-simple unsigned32="0" hexanum="0x0" />
```
Comment (pedromartinez) [inline lib/conduit/marben/views/reauthorization_request.erb]: What different kinds of request types can go here?

File: lib/conduit/marben/actions/reauthorization_request.rb (hunk 1 above)
Comment (captainbeardo) [inline lib/conduit/marben/actions/reauthorization_request.rb]: https://www.youtube.com/watch?v=fsbiQSRUhsU

Comment (pedromartinez) [inline lib/conduit/marben/actions/reauthorization_request.rb]: That was not helpful.

Comment (captainbeardo) [inline lib/conduit/marben/actions/reauthorization_request.rb]: Just stream of consciousness. That's what I thought of when I typed `super`.

Comment (pedromartinez) [inline lib/conduit/marben/actions/reauthorization_request.rb]: Oh, haha. I get the joke. I was wondering why we had to define the
initialize method here if all we are calling is super.

On Wednesday, February 10, 2016, beardo <notifications@github.com> wrote:

> In lib/conduit/marben/actions/reauthorization_request.rb
> <https://github.com/conduit/conduit-marben/pull/4#discussion_r52536324>:
>
> > @@ -0,0 +1,13 @@
> > +require 'conduit/marben/actions/base'
> > +
> > +module Conduit::Driver::Marben
> > +  class ReauthorizationRequest < Base
> > +    required_attributes :message_ete, :message_hbh, :session_id,
> > +      :origin_host, :origin_realm, :destination_realm, :destination_host
> > +
> > +    def initialize(options = {})
> > +      super # thanks for asking...
>
> Just stream of consciousness. That's what I thought of when I typed super.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conduit/conduit-marben/pull/4/files#r52536324>.
>


-- 

Pedro Martinez
Director, Product Development
Hello Labs
561-451-6630

Comment (captainbeardo) [inline lib/conduit/marben/actions/reauthorization_request.rb]: It was there from when I lifted code from another action that did do something different other than just super.

**Pull request merged**